appscale/tools/parse_args.py

```diff
@@ -183,6 +187,10 @@ def add_allowed_flags(self, function):
     #TODO: remove arguments in appscale-run-instances that are no longer
     # supported. (min, max, appengine, max_memory, scp)
     if function == "appscale-run-instances":
+      self.parser.add_argument('--update',
+        nargs='+',
+        choices=self.ALLOWED_DIR_UPDATES,
+        default="", help="updates specified code directory and builds it")
```

Comment:
I would recommend an empty list as a default so that you can be sure the parsed arg object is a list.